### ISSUE TITLE
Add multiple Docker networks support

### ISSUE THREAD

**Issue by emilevauge** [containous/traefik#229]
As seen in this in #227, Traefik should support multiple docker networks.

**Comment by avthart** [comment]
I am using traefik with Docker Compose for local development. Docker Compose will create a new network by default for your docker-compose file. Therefore it would be nice if traefik supports multiple docker networks.

**Comment by maljac** [comment]
+1

**Comment by sebmoule** [comment]
I have a swarm cluster with containers that are mapped so several overlay networks.

When Traefik retrieve it's conf from the swarm master it concatene all the IP result that it won't works:

```json
   "backend-test": {
      "servers": {
        "server-seb_test_1": {
          "weight": 1,
          "url": "http://192.168.6.2192.168.8.2:80"
        }
      }
    },
```

my container have 2 ip 192.168.6.2 & 192.168.8.2

is there a way to specify traefik which network to use in this case somethink like **traefik.network** ?

```
    labels:
      - "traefik.backend=test"
      - "traefik.port=80"
      - "traefik.frontend.rule=Host:test.example.com"
      - "traefik.network=net_front"  
    networks:
      - net_front
      - net_back_api
      - net_back_tool

```

**Comment by xt99** [comment]
+1

**Comment by tbouvet** [comment]
+1

I have a swarm cluster with containers that are mapped on several overlay networks.
```
version: '2'
services:
  myapp:
    expose:
      - "8080"
    depends_on:
      - mongodb
    labels:
      - "traefik.backend=my-app"
      - "traefik.frontend.rule=Host:my-app.docker.localhost"
    networks:
      - front
      - application-backend
      
  mongodb:
    image: mongo:latest
    container_name: mongodb
    networks:
      - application-backend
    
networks:
  application-backend:
    driver: overlay
  front:
    external:
      name: traefik_front
```

When Traefik retrieve the conf from the swarm master, there is IP literal concatenation.
See the debul log:
```
traefik    | time="2016-05-31T08:49:14Z" level=debug msg="Creating server server-compose_my-app_1 at http://10.0.3.210.0.1.310.0.0.3:8080 with weight 1"
```

**Comment by emilevauge** [comment]
Fixed by #415
